lib/src/controls/form/color_picker/color_picker.dart

```diff
@@ -0,0 +1,1143 @@
+                const SizedBox(height: ColorPickerSpacing.large),
+                _buildMoreButton(),
+              ],
+              if (!widget.isMoreButtonVisible || _isMoreExpanded) ...[
+                if (hasVisibleInputs) ...[
+                  const SizedBox(height: ColorPickerSpacing.large),
+                  _buildInputs(),
+                ],
+              ],
+            ],
+          )
+        : Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              _buildSpectrumAndPreview(),
+              if (widget.isColorSliderVisible ||
+                  (widget.isAlphaEnabled && widget.isAlphaSliderVisible)) ...[
+                const SizedBox(width: ColorPickerSpacing.large),
+                _buildSliders(),
+              ],
+              if (hasVisibleInputs) ...[
+                const SizedBox(width: ColorPickerSpacing.large),
+                _buildInputs(),
+              ],
+            ],
+          );
+  }
+
+  /// Initializes the text controllers and focus nodes.
+  void _initControllers() {
+    _hexController = TextEditingController();
+    _hexFocusNode = FocusNode();
+    _hexFocusNode.addListener(_onFocusChange);
+  }
+
+  /// Updates the text controllers with the current color state.
+  void _updateControllers() {
+    final oldText = _hexController.text;
+    final newText = _colorState.toHexString(widget.isAlphaEnabled);
+    if (oldText != newText) {
+      _hexController.text = newText;
+    }
+  }
+
+  /// Builds the color spectrum and color preview box.
+  Widget _buildSpectrumAndPreview() {
+    return _ColorSpectrumAndPreview(
+      colorState: _colorState,
+      orientation: widget.orientation,
+      colorSpectrumShape: widget.colorSpectrumShape,
+      isColorPreviewVisible: widget.isColorPreviewVisible,
+      onColorChanged: _handleColorChanged,
+      minHue: widget.minHue,
+      maxHue: widget.maxHue,
+      minSaturation: widget.minSaturation,
+      maxSaturation: widget.maxSaturation,
+    );
+  }
+
+  /// Builds the color sliders.
+  Widget _buildSliders() {
+    return _ColorSliders(
+      colorState: _colorState,
+      orientation: widget.orientation,
+      isColorSliderVisible: widget.isColorSliderVisible,
+      isAlphaSliderVisible: widget.isAlphaSliderVisible,
+      isAlphaEnabled: widget.isAlphaEnabled,
+      onColorChanged: _handleColorChanged,
+      minValue: widget.minValue,
+      maxValue: widget.maxValue,
+    );
+  }
+
+  /// Builds the "More" button to expand the color picker inputs.
+  Widget _buildMoreButton() {
+    final moreButton = SizedBox(
+        width: ColorPickerSizes.inputBox,
+        child: Button(
+          onPressed: () => setState(() => _isMoreExpanded = !_isMoreExpanded),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(_isMoreExpanded ? 'Less' : 'More'),
```

Comment:
Add a TODO to add these to localization later

Reply:
added the TODO comment